src/styles/index.scss

```diff
@@ -1,3 +1,102 @@
-body {
+@import './variables';
+
+.page {
+  height: 100%;
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  font-family: Arial, sans-serif;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: calc(7 * #{$calendar-size} + 6 * #{$calendar-gap});
```

Comment:
I would recommend extracting the calculation `7 * #{$calendar-size} + 6 * #{$calendar-gap}` into a separate variable for better readability and maintainability. It's a code quality issue.